src/SearchResults.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import moment from "moment";
+
+const SearchResults = props => {
+  //created to calculated the length of stay, any idea for the improvments??
+
+  // const checkTheLengthOfTheStay = props.results.map((element) => {
```

Comment:
Could you remove any comments code if you don't need it?

Reply:
Correction done
Thanks Chris